### PR TITLE
Moving cluster admins to sudoers group 

### PR DESCRIPTION
**Pull request by dystewart**
In response to [https://github.com/operate-first/support/issues/696](https://github.com/operate-first/support/issues/696)

Adding sudoers group to all clusters and removing cluster-admins group from clusters.

### CODE REVIEW

**Comment by HumairAK** [comment]
as this affects emea/moc/osc, can we get a review from: 
@larsks , @rbo , @redmikhail

**Comment by HumairAK** [comment]
@dystewart  can you also add docs on how to run cli commands + how to continue gaining console access as cluster admin

File: docs/content/cluster-scope/obtain_cluster_admin_access.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Obtain Cluster Admin Priviledges in an Operate First Cluster
```
Comment (HumairAK) [inline docs/content/cluster-scope/obtain_cluster_admin_access.md:1]: > Priviledges

to 

privileges

File: docs/content/cluster-scope/obtain_cluster_admin_access.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Pre-requisites
+
+1. Be a member of the sudoers group in the desired cluster
+2. Have access to the desired cluster via the CLI or the OpenShift
```
Comment (HumairAK) [inline docs/content/cluster-scope/obtain_cluster_admin_access.md:6]: What sort of access does one need to do impersonation from the console?

File: cluster-scope/overlays/prod/emea/demo/kustomization.yaml [hunk 3]
```diff
@@ -4,14 +4,15 @@ kind: Kustomization
 resources:
 - ../common
 - ../../../../base/config.openshift.io/oauths/cluster
-- ../../../../base/user.openshift.io/groups/cluster-admins
 - ../../../../base/core/configmaps/cluster-monitoring-config
 - ../../../../base/core/namespaces/opf-alertreceiver
 - ../../../../base/operators.coreos.com/subscriptions/opendatahub-operator
 - ../../../../base/core/namespaces/demo-jupyterhub
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/sudoers
+- ../../../../base/user.openshift.io/groups/sudoers
```
Comment (HumairAK) [inline cluster-scope/overlays/prod/emea/demo/kustomization.yaml:12]: looks like you always include these together, can you create a bundle with these files?

File: cluster-scope/overlays/prod/osc/osc-cl1/groups/sudoers.yaml [hunk 4]
```diff
@@ -1,11 +1,10 @@
 apiVersion: user.openshift.io/v1
 kind: Group
 metadata:
-  name: cluster-admins
+  name: sudoers
```
Comment (HumairAK) [inline cluster-scope/overlays/prod/osc/osc-cl1/groups/sudoers.yaml:4]: by removing us as cluster-admins, what sort of permissions do we require to be able to impersonate this group from the console? are these already available?

File: docs/content/cluster-scope/obtain_cluster_admin_access.md (hunk 2 above)
Comment (dystewart) [reply docs/content/cluster-scope/obtain_cluster_admin_access.md:6]: To do impersonation from the console users need cluster-admin rights, unless they are a sudoer

**Review by larsks — APPROVED** [review]

**Pull request closed without merging**